srcpkgs/python-xcffib/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'python-xcffib'
+pkgname=python-xcffib
+version=0.8.1
+revision=1
+wrksrc="${pkgname#*-}-${version}"
+build_style=python-module
+pycompile_module="${pkgname#*-}"
+_pyhost=(python{,3}-{devel,setuptools})
+hostmakedepends="${_pyhost[@]} pkg-config cabal-install parallel xcb-proto"
+makedepends="python-devel python-six python-cffi libffi-devel"
+depends="python-six python-cffi"
+short_desc="Drop-in replacement for xpyb based on cffi"
+maintainer="Mohammad Amin Sameti <mamins1376@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/tych0/xcffib"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=879d9bc360e63e62a8f9b80299127ecbdb7de90271ebfe6a532fbb894efab59a
+
+pre_build() {
+	cabal v2-update
+	PKG_CONFIG_PATH=/usr/lib/pkgconfig make xcffib
+}
+
+python3-xcffib_package() {
+	makedepends="${makedepends//python/python3}"
```

Comment:
Only depends are meaningful in subpkgs. There has to be pkg_install defined, and preferably short_desc.